kctf-operator/config/crd/bases/kctf.dev_challenges.yaml

```diff
@@ -0,0 +1,7359 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
and this is another dev_challenges file?

Reply:
oh, this is a base file, this one id different. here we can make changes that then get propagated up to the final one.

Reply:
actually this one is generated too it seems.

Reply:
https://github.com/google/kctf/blob/3a2f7ec42ee47e9d21080a3e43520c2dc62f4323/kctf-operator/Makefile#L108
https://github.com/google/kctf/blob/3a2f7ec42ee47e9d21080a3e43520c2dc62f4323/kctf-operator/Makefile#L77

looks like these are generated by different rules. we could remove them I think :grimacing:

Reply:
nope we can't haha.. this broke everything